src/main/java/duke/Task.java

```diff
@@ -0,0 +1,65 @@
+package duke;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
```

Comment:
Nice use of variable name "isDone" rather than "done" for the bool variable name👍 It was a common mistake.